lib/discover/hyperbahn.thrift

```diff
@@ -0,0 +1,35 @@
+exception NoPeersAvailable {
```

Comment:
I don't think it will be a drop in replacement if the `hyperbahn.thrift` file needs to be updated, meaning we need to change code in the discover file for hyperbahn as well.